datadog_checks_dev/datadog_checks/dev/tooling/commands/ci/setup.py

```diff
@@ -0,0 +1,62 @@
+# (C) Datadog, Inc. 2019-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+import platform
+import subprocess
+
+import click
+
+from datadog_checks.dev.tooling.constants import get_root
+from datadog_checks.dev.tooling.testing import get_tox_envs
+
+from ..console import CONTEXT_SETTINGS
+
+PLATFORM = platform.system().lower()
+SCRIPTS_PATH = os.path.join(get_root(), '.azure-pipelines', 'scripts')
+
+
+def display_action(script_file):
+    display_header = f'Running: {script_file}'
+    print(f'\n{display_header}\n{"-" * len(display_header)}\n')
+
+
+@click.command(context_settings=CONTEXT_SETTINGS, short_help='Run CI setup scripts')
+@click.argument('checks', nargs=-1)
+@click.option('--changed', is_flag=True, help='Only target changed checks')  # Added for compatibility with `ddev test`
```

Comment:
`--changed` added for compatibility, otherwise this input will fail: https://github.com/DataDog/integrations-core/blob/master/.azure-pipelines/changes.yml#L24